pennylane_lightning/src/simulator/Measures.hpp

```diff
@@ -318,32 +318,35 @@ class Measures {
      * between metropolis states.
      * @param gen Random number generator.
      * @param distrib Random number distribution.
-     * @param s1 initial state
+     * @param init_idx Init index of basis state.
      */
     size_t metropolis_step(const SVType &sv,
                            std::unique_ptr<TransitionKernel<fp_t>> &tk,
```

Comment:
Since this is coming as a unique_ptr, it doesn't need to be a reference.

Reply:
Good spot! Thanks @mlxd !

Reply:
Do you mean to use `std::move` transfer ownership of `tk` to `metropolis_step`?

Reply:
Yep, assuming it is safe to do so. 

Though, in hindsight, I am now wondering if that would be a good idea. I think not, so forgot I suggested this. Sorry for the confusion.

Reply:
Fine to keep as is, but maybe make it a `const` reference since it won't be modified.

Reply:
yea, I will make it a const reference. need to pass tk for many iterations, so will keep it as it is.